test/sql/test_conf/T/test_update_starlet_config

```diff
@@ -0,0 +1,13 @@
+-- name: test_update_starlet_config
```

Comment:
case shall be tagged as `@cloud` to ensure they are only valid in shared-data cluster.

Reply:
done